infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutAlignmentDao.kt

```diff
@@ -169,10 +169,6 @@ class LayoutAlignmentDao(
             .associate { it }
     }
 
-    @Transactional
```

Comment:
Kirjoittelin nämä aikanaan ajatuksella että noodeja ja edgejä voisi tallennella myös ihan muuten vain. Eihän sellaista toki mistään tehdä tuo transaktional-annotaatiokaan ei koskaan toimi koska ei tätä ulkoa kutsuta. 

Alkuperäinen ajatusmalli oli myös että ne oliot mapataan aina db-olioiksi ennen käyttöä, mikä on noop jos se on jo db-olio, tallentuu muutoin. Koska näitä käytetään vain raidegeometrian tallennuksessa päätin purkaa tuon koko härvelin pois ja vain suodattaa ne Tmp-oliot ja tallentaa ne, poimien niiden id:t eteenpäin seuraavaan tallennukseen. 

Koodina ei minusta isoa eroa nättiyden suhteen, mutta tällä rakenteella olisi helpompi tehdä ne bulk-insertit nodeille ja edgeille. Ne ei kuitenkaan sisälly tähän pullariin, koska niissä on muutakin ongelmaa ratkaistavaksi.